binary_search_tree/binary_search_tree.py

```diff
@@ -5,13 +5,50 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+      if value < self.value:
+          if not self.left:
+              self.left = BinarySearchTree(value)
+          else:
+              self.left.insert(value)
+      elif value >= self.value:
+          if self.right is None:
+              self.right = BinarySearchTree(value)
+          else:
+              self.right.insert(value)
 
   def contains(self, target):
-    pass
+        if target < self.value:
+            if self.left is None:
+                return False
+            return self.left.contains(target)
+        elif target > self.value:
+            if self.right is None:
+                return False
+            return self.right.contains(target)
+        else:
+            return True
 
   def get_max(self):
-    pass
+    current = self
+    while current.right:
+      current = current.right
+    return current.value
 
```

Comment:
Nice work utilizing the structure of BST to create efficient methods for contains and get_max.